fmt/src/git.rs

```diff
@@ -34,38 +38,45 @@ impl GitHelper {
             return Ok(None);
         }
 
-        match Repository::discover(basedir) {
+        match gix::open(basedir) {
```

Comment:
You probably want to stick to `gix::discover()` here, otherwise it's not the same, `open` can only see what's in the given directory, but won't look upwards.

Reply:
Yes exactly. Thank you!